python/demos/Non_Stationary_Exploration.ipynb

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
Does it make sense to move this notebook to `python/docs/source/examples` so that it can be rendered on the website?

Reply:
This notebook is pretty experimental, but I can clean it up then move it

Reply:
I think as long as the notebook specifies that it's fine